### ISSUE TITLE
Right alt is not set as AltGr in the US intl. AltGr layout

### ISSUE THREAD

**Issue by arael** [fcitx/fcitx5#775]
The "English (US) - English (intl., with AltGr dead keys)" layout does not set the Right Alt key as AltGr.

1. Set the "English (US) - English (intl., with AltGr dead keys)"
2. Press RAlt+` a

**Expected behavior**
The sequence RAlt+` a should produce:  **à**.
Instead what I get is **`a**

**Desktop (please complete the following information):**
 - Desktop: Sway
 - Display server type: Wayland

The input works correctly if I do not use fcitx5. But in Sway other input systems have issues.

**Comment by wengxt** [comment]
it’s more relevant to there’s no generic way to configure layout under wayland. 

Did you set the layout in sway config?

**Comment by arael** [comment]
I did set the layout:

input "Keychron Keychron Q1" {
    xkb_layout us
    xkb_variant ,altgr-intl
}

And it is probably a wayland issue because the same keyboard, without any additional configuration, has a functional AltGr in i3 while using fcitx5.

**Comment by arael** [comment]
There was a problem with my Sway config file. Thank you for pointing me in the right direction.

This worked:
```
input "13364:263:Keychron_Keychron_Q1_Keyboard" {
    xkb_layout us
    xkb_variant altgr-intl
}
```

**Comment by wengxt** [comment]
I don’t think your config is correct. There’s an additional comma.

It should either be “us,us” “,altgr-intl”

or us altgr-intl.

Do you use intend to use the non intl layout?

**Comment by arael** [comment]
Indeed, you are right. I saw your last comment only after submitting my comment. Thank you very much helping me.
